example/otel-collector/k8s/jaeger/jaeger.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: jaegertracing.io/v1
```

Comment:
```suggestion
# Copyright The OpenTelemetry Authors
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

apiVersion: jaegertracing.io/v1
```

Reply:
I'm not sure that it's appropriate to put our license header on the Jaeger operator resource definitions.  If policy requires our license headers on all files perhaps we should reference the original location of these files.

@lizthegrey do you have insight/opinions on this? I don't want to get too far out over my skis on this, but it does set off my spidey sense.

Reply:
If this was copied from another source you are correct, we need to attribute it to that source.

Reply:
The Jaeger files have been taken from [here](https://github.com/jaegertracing/jaeger-operator/tree/master/deploy). I just made some changes because the namespaces were inconsistent. Do we still want the licence header in this case?